### PR TITLE
Add 1.5.1.0rc2 release notes

### PR DESCRIPTION
**Pull request by sickpig**

### CODE REVIEW

File: doc/release-notes-bucash.md [hunk 1]
```diff
@@ -38,8 +38,17 @@ Other than that upgrading from a version lower than 1.5.0.1 your client is proba
 on a minority chain and need some manual intervention to make so that it follow the majority
 chain after the upgrade. For more detail please look at `reconsidermostworkchain` RPC commands.
 
-Main Changes
-------------
+Bug Fixed in 1.5.1.0-rc2
+------------------------
+
+- Making Xversion more permission in the initial handshake
```
Comment (gandrewstone) [inline doc/release-notes-bucash.md]: permission -> permissive

File: doc/release-notes-bucash.md [hunk 2]
```diff
@@ -38,8 +38,17 @@ Other than that upgrading from a version lower than 1.5.0.1 your client is proba
...
+- Set EB properly while in SV mode
```
Comment (gandrewstone) [inline doc/release-notes-bucash.md]: EB -> Excessive Block Size (EB)

File: doc/release-notes-bucash.md [hunk 3]
```diff
@@ -38,8 +38,17 @@ Other than that upgrading from a version lower than 1.5.0.1 your client is proba
...
+- Ensure that blk files storage operate on an append-only basis
+- Various minor fixes/improvements to thwe python QA framework
```
Comment (gandrewstone) [inline doc/release-notes-bucash.md]: thwe -> the

File: doc/release-notes-bucash.md [hunk 4]
```diff
@@ -38,8 +38,17 @@ Other than that upgrading from a version lower than 1.5.0.1 your client is proba
...
+- Reduce the last of thinblock/preferential timer to 1 sec from 10 sec.
+
```
Comment (gandrewstone) [inline doc/release-notes-bucash.md:50]: Add:

Fixed connection drop bug where a ping that happened soon after initial connection would be ignored, causing an eventual ping timeout.

**Review by gandrewstone — CHANGES_REQUESTED** [review]

File: doc/release-notes-bucash.md (hunk 1 above)
Comment (sickpig) [reply doc/release-notes-bucash.md]: will fix

File: doc/release-notes-bucash.md (hunk 2 above)
Comment (sickpig) [reply doc/release-notes-bucash.md]: will fix

File: doc/release-notes-bucash.md (hunk 3 above)
Comment (sickpig) [reply doc/release-notes-bucash.md]: will fix

File: doc/release-notes-bucash.md (hunk 4 above)
Comment (sickpig) [reply doc/release-notes-bucash.md:50]: will do

**Review by gandrewstone — APPROVED** [review]

**Pull request merged**